engineering_tools/vector_map_scripts/osm_transform.py

```diff
@@ -0,0 +1,79 @@
+import argparse
+from lxml import etree
+from pyproj import CRS, Transformer
+import math
+import numpy as np
+
+# === Parse command-line arguments ===
+parser = argparse.ArgumentParser(description="Transform and rotate an OSM map based on geoReference.")
+parser.add_argument("input_file", help="Path to the input .osm file")
+parser.add_argument("output_file", help="Path to the output .osm file")
+args = parser.parse_args()
+
+# === Fixed rotation angle ===
+rotation_deg = 0  # Counter-clockwise
```

Comment:
Can we add a `# TODO` comment to rotation_deg to highlight that the user should adjust this?